packages/next/src/build/webpack-config.ts

```diff
@@ -911,6 +911,16 @@ export default async function getBaseWebpackConfig(
   const builtinModules = (require('module') as typeof import('module'))
     .builtinModules
 
+  // Add Bun builtins to externals
+  const bunExternals = [
+    'bun:ffi',
+    'bun:jsc',
+    'bun:sqlite',
+    'bun:test',
+    'bun:wrap',
+    'bun',
+  ]
```

Comment:
Shouldn't this be solved by `bun` implementing [`module.builinModules`](https://nodejs.org/api/module.html#modulebuiltinmodules)?

Reply:
That seems dangerous. You'd be prerendering with Node.js but then using Bun for dynamic requests.